extensions/podman/src/util.ts

```diff
@@ -38,6 +38,11 @@ export function appHomeDir(): string {
   return xdgDataDirectory + '/podman';
 }
 
+const configDirectory = '.config/containers';
+export function appConfigDir(): string {
+  return `${configDirectory}/podman`;
```

Comment:
You only seems to be calling this method inside isMac condition, but this is a valid directory for other platform, maybe use path.join ?

Reply:
I mimic previous method

so here it returns a fragment of a directory and not a full path

It means that you need to use with path methods to resolve it (like path.resolve)

And we can use slashes in path methods

Reply:
so it's valid even for Windows (like the previous method `appHomeDir` is